subgraph/fluence-graph.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+# Deploy on FluenceLabs stand (Currently only on stage).
+#
+# Example Run:
+# >> export BASIC_AUTH_FLUENCE_STAGE=<check .example.env> && ./fluence-graph.sh
+
+# Args:
+# $1 - action {deploy, create}.
+
+# Envs:
+# - BASIC_AUTH_FLUENCE_STAGE
+
+# Validate envs.
+if [ "$BASIC_AUTH_FLUENCE_STAGE" = "" ]; then
+  echo "Please provide 'BASIC_AUTH_FLUENCE_STAGE' env."
+  exit 0
+fi
+
+# Choose auth string.
+basic_auth=""
+if [ "$BASIC_AUTH_FLUENCE_STAGE" != "" ]; then
```

Comment:
```suggestion
if [[ -z $BASIC_AUTH_FLUENCE_STAGE ]]; then
```

`[[` is bash builtin command, `[` is legacy command outside of bash, if I'm not mistaken.

It is a minor thing but according to modern bash guidelines it is better to use `[[` :)
http://mywiki.wooledge.org/BashGuide/TestsAndConditionals

`-z` check means `var is undefined or empty`

Reply:
even for strings?